projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-layout.php

```diff
@@ -130,10 +132,12 @@ private function template( $name, $context = null ) {
 	/**
 	 * Handle tiled gallery partial path.
 	 *
+	 * @html-template-var array $context
+	 *
 	 * @param string $name - the name.
 	 * @param array  $context Context array passed to the partial.
 	 */
-	private function partial( $name, $context = null ) {
```

Comment:
I was going to see if this and the next default param change affected the filter docs, but it doesn't look like the filters are documented. :^)